source/RunScribeDataField.mc

```diff
@@ -151,24 +167,24 @@ class RunScribeDataField extends Ui.DataField {
     
     hidden function getMetricName(metricType) {
         if (metricType == 1) {
-            return "Impact Gs";
+            return "Impact Gs"; // Ui.loadResource(Rez.Strings.impact_gs_app_label);
         } else if (metricType == 2) {
-            return "Braking Gs";
+            return "Braking Gs"; // Ui.loadResource(Rez.Strings.braking_gs_app_label);
         } else if (metricType == 3) {
-            return "Footstrike";
+            return "Footstrike"; // Ui.loadResource(Rez.Strings.fs_type_app_label);
         } else if (metricType == 4) {
-            return "Pronation";
+            return "Pronation"; // Ui.loadResource(Rez.Strings.pronation_app_label);

```

Comment:
Please, remove comments here, above and below + add units for Pronation.

Reply:
No space for units on all screen sizes.